x/feemarket/keeper/keeper.go

```diff
@@ -52,6 +53,89 @@ func (k *Keeper) GetAuthority() string {
 	return k.authority
 }
 
+// GetFeeDenomParam returns the feeDenomParam for feeDenom.
+func (k *Keeper) GetFeeDenomParam(ctx sdk.Context, feeDenom string) (types.FeeDenomParam, error) {
+	store := ctx.KVStore(k.storeKey)
+
+	key := types.GetKeyPrefixFeeDenomParam(feeDenom)
+	bz := store.Get(key)
+
+	fdp := types.FeeDenomParam{}
+	if bz == nil {
+		return fdp, sdkerrors.ErrKeyNotFound.Wrapf("feeDenomParam not found for feeDenom: %s", feeDenom)
+	}
+
+	if err := fdp.Unmarshal(bz); err != nil {
+		return fdp, err
+	}
```

Comment:
Should we better return `nil, err` instead of the empty type and error? nil can save someone good time of debugging when not checking the error